nikola/plugins/command/plugin.py

```diff
@@ -229,104 +217,84 @@ def do_install(self, url, name, show_install_notes=True):
             LOGGER.info('Installing dependencies with pip...')
             try:
-                subprocess.check_call((sys.executable, '-m', 'pip', 'install', '-r', reqpath))
+                subprocess.check_call((sys.executable, '-m', 'pip', 'install', '-r', str(requirements_path)))
             except subprocess.CalledProcessError:
                 LOGGER.error('Could not install the dependencies.')
                 print('Contents of the requirements.txt file:\n')
-                with io.open(reqpath, 'r', encoding='utf-8-sig') as fh:
-                    print(utils.indent(fh.read(), 4 * ' '))
-                print('You have to install those yourself or through a '
-                      'package manager.')
+                print(utils.indent(requirements_path.read_text(), 4 * ' '))
+                print('You have to install those yourself or through a package manager.')
             else:
                 LOGGER.info('Dependency installation succeeded.')
 
-        reqnpypath = os.path.join(dest_path, 'requirements-nonpy.txt')
-        if os.path.exists(reqnpypath):
-            LOGGER.warning('This plugin has third-party '
-                           'dependencies you need to install '
-                           'manually.')
+        requirements_nonpy_path = dest_path / 'requirements-nonpy.txt'
+        if requirements_nonpy_path.exists():
+            LOGGER.warning('This plugin has third-party dependencies you need to install manually.')
             print('Contents of the requirements-nonpy.txt file:\n')
-            with io.open(reqnpypath, 'r', encoding='utf-8-sig') as fh:
-                for l in fh.readlines():
-                    i, j = l.split('::')
-                    print(utils.indent(i.strip(), 4 * ' '))
-                    print(utils.indent(j.strip(), 8 * ' '))
-                    print()
+            for l in requirements_nonpy_path.read_text().strip().split('\n'):
+                i, j = l.split('::')
+                print(utils.indent(i.strip(), 4 * ' '))
+                print(utils.indent(j.strip(), 8 * ' '))
+                print()
 
             print('You have to install those yourself or through a package '
                   'manager.')
 
-        req_plug_path = os.path.join(dest_path, 'requirements-plugins.txt')
-        if os.path.exists(req_plug_path):
+        requirements_plugins_path = dest_path / 'requirements-plugins.txt'
+        if requirements_plugins_path.exists():
             LOGGER.info('This plugin requires other Nikola plugins.')
             LOGGER.info('Installing plugins...')
             plugin_failure = False
             try:
-                with io.open(req_plug_path, 'r', encoding='utf-8-sig') as inf:
-                    for plugname in inf.readlines():
-                        plugin_failure = self.do_install(url, plugname.strip(), show_install_notes) != 0
+                for plugin_name in requirements_plugins_path.read_text().strip().split('\n'):
```

Comment:
Why not use `splitlines()`? That should also work for non-Unix line endings.
```suggestion
                for plugin_name in requirements_plugins_path.read_text().strip().splitlines():
```